### PR TITLE
Removed PUP references in the docs.

### PR DESCRIPTION
**Pull request by ipanova**
closes #7747

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Review by mdellweg — APPROVED** [review]

File: docs/contributing/index.rst [hunk 1]
```diff
@@ -54,10 +54,7 @@ Suggesting Changes to the Pulp Development Process
...
 Pulp is a community project, and major changes to the way Pulp is developed, such as the release
-cycle, and style guide, can be suggested by submitting a :term:`PUP`, or, "Pulp Update Proposal".
-All approved PUPs live in the `PUP repository <https://github.com/pulp/pups/>`_.
+cycle, and style guide, can be proposed:
 
-`PUP-1 <https://github.com/pulp/pups/blob/master/pup-0001.md>`_ defines the PUP process itself.
-
-All approved, rejected, and abandoned PUPs are tracked in the `PUP index
-<https://github.com/pulp/pups/blob/master/README.md>`_
+  * at the **Open Floor** - that takes place every Tuesday and Friday 10:30 ET (either EST or EDT) in #pulp-meeting on Freenode (IRC)
```
Comment (goosemania) [inline docs/contributing/index.rst:59]: Do we have a place where we already share when open floor happens? Maybe on our website, can we link to there?
I'm worried to put such info in multiple places, we'll forget to update it.

Comment (goosemania) [reply docs/contributing/index.rst:59]: https://pulpproject.org/get_involved/#meetings

Comment (ipanova) [reply docs/contributing/index.rst]: I understand and share your concern, however I specifically wanted to highlight 2 ways on how to propose the process changes, in the meantime `get_involved` contains much more information and not quite related. Suggestions on phrasing are welcome.

Comment (mdellweg) [reply docs/contributing/index.rst]: I agree, that it's better not to include the times for open floor. :+1:

Comment (goosemania) [reply docs/contributing/index.rst]: > I understand and share your concern, however I specifically wanted to highlight 2 ways on how to propose the process changes, in the meantime `get_involved` contains much more information and not quite related. Suggestions on phrasing are welcome.

```
* at the **Open Floor** , see the schedule `here <https://pulpproject.org/get_involved/#meetings>`_.
```
?

**Review by goosemania — APPROVED** [review]

**Pull request merged**